Source/WTF/wtf/Assertions.h

```diff
@@ -249,10 +250,8 @@ WTF_EXPORT_PRIVATE void WTFPrintBacktrace(void** stack, int size);
 WTF_EXPORT_PRIVATE void WTFReleaseLogStackTrace(WTFLogChannel*);
 #endif
 
-#if USE(ODH_TELEMETRY)
-#define REPORT_ODH_ERROR(file, line, func, ...)    WTFReportOdhError(file, line, func, __VA_ARGS__)
-WTF_EXPORT_PRIVATE void WTFReportOdhError(const char* file, int line, const char* function, const char* format, ...);
-WTF_EXPORT_PRIVATE void WTFReportOdhErrorV(const char* file, int line, const char* function, const char* format, va_list args);
+#if USE(RDK_TELEMETRY)
+#define REPORT_ODH_ERROR(file, line, func, ...)    Telemetry::reportError(file, line, func, __VA_ARGS__)
 #else
 #define REPORT_ODH_ERROR(file, line, func, ...)    ((void)0)
```

Comment:
There's no need for this ifdefology, REPORT_ODH_ERROR should be defined the same no matter if we're using rdk telemetry or not. Telemetry::reportError function exists in both cases (though it's empty in non-rdk case)